bitrise.yml

```diff
@@ -1,278 +1,67 @@
 
-  test:
-    after_run:
-    # headless tests first, as otherwise running emulator is reused
-    - ci-objc_headless_mode
-    - _script_wrapper_for_headless_failing_ui_test
-    - ci-objc_headless_mode_xcpretty
-    - ci-parallel_headless_mode
-    - tvos_headless_mode
-
-  debug:
-    envs:
-    # Remote or local URL
-    - SAMPLE_APP_URL: $SAMPLE_APP_URL
-    - SAMPLE_APP_BRANCH: $SAMPLE_APP_BRANCH # main
-    - BITRISE_PROJECT_PATH: $BITRISE_PROJECT_PATH
-    - BITRISE_SCHEME: $BITRISE_SCHEME
-    - SIMULATOR_DEVICE: $SIMULATOR_DEVICE # iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: $SIMULATOR_OS_VERSION # latest
-    - SIMULATOR_PLATFORM: $SIMULATOR_PLATFORM # iOS Simulator
-    - OUTPUT_TOOL: xcpretty
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "yes"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  manual_test-retry_after_fail:
+  sample:
     envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: failing-ui-test
-    - OUTPUT_TOOL: xcpretty
     - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
     - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "yes"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  ci-objc_headless_mode:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcodebuild
-    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
-    - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-    - check_exported_artifacts
-
-  ci-objc_headless_mode_xcpretty:
-    envs:
-    - SAMPLE_APP_URL: https://github.com/bitrise-io/sample-apps-ios-simple-objc-with-uitest.git
-    - SAMPLE_APP_BRANCH: master
-    - OUTPUT_TOOL: xcpretty
-    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
-    - BITRISE_SCHEME: ios-simple-objc
-    - SIMULATOR_DEVICE: iPhone 8 Plus
-    - SIMULATOR_OS_VERSION: "latest"
-    - SIMULATOR_PLATFORM: iOS Simulator
-    - HEADLESS_MODE: "yes"
-    - RETRY_ON_FAIL: "no"
-    after_run:
-    - _test
-    - check_exported_artifacts
-    - check_xcpretty_html_report
-
-  _script_wrapper_for_headless_failing_ui_test:
     steps:
     - script:
-        title: Start a failing workflow, wrapped in a script.
         inputs:
         - content: |-
             #!/bin/env bash
-            bitrise run ci-objc_headless_mode_failing_ui_test
-            if [ $? -ne 1 ] ; then
-              echo "Workflow was excepted to fail, exit code not 1."
-              exit 1
-            fi
+            set -ex
+            rm -rf ./_tmp
+    - change-workdir:
```

Comment:
change-workdir is not necessarily needed: https://github.com/bitrise-steplib/steps-xcode-archive/blob/STEP-888-new-bitrise-yml/bitrise.yml#L65-L82

Reply:
Udpated